README.md

```diff
@@ -1,34 +1,28 @@
+#### What is SpatialOS GDK for Unreal Engine?
+The SpatialOS GDK for Unreal Engine is an Unreal Engine 4 (UE4) [plugin](https://docs.unrealengine.com/en-us/Programming/Plugins) made by Improbable. You can use the GDK to make use of multiple Unreal servers in a single game instance using the SpatialOS platform, unlocking the ability to implement large scale, complex and persistent worlds. Find out more about the GDK in [our blogpost](https://improbable.io/games/blog/spatialos-unreal-gdk-pre-alpha).
 
 
 * License: use of the contents of this repository is subject to the [Improbable license](https://docs.improbable.io/unreal/latest/license) </br>
 (Your access to and use of the Unreal Engine is governed by the [Unreal Engine End User License Agreement](https://www.unrealengine.com/en-US/previous-versions/udk-licensing-resources?sessionInvalidated=true). Please ensure that you have agreed to those terms before you access or use the Unreal Engine.)
-* Version: pre-alpha (for evaluation only)
-* Unreal Game Development Kit repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* Version: Alpha (Stability and performance improvements pending)
+* GDK Repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 
 # Documentation
-* For information on the SpatialOS GDK for Unreal, we recommend you use the [GDK documentation on the SpatialOS website](https://docs.improbable.io/unreal/pre-alpha/index).
+* For information on the GDK, we recommend you use the [GDK documentation on the SpatialOS website](https://docs.improbable.io/unreal/pre-alpha/index).
 * For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). <br/>
 
 ## Recommended use
-We are releasing the GDK in [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) so we can react to feedback and iterate on development quickly. To facilitate this, during our pre-alpha stage we don't have a formal deprecation cycle for APIs and workflows. This means that everything and anything can change. In addition, documentation is limited and some aspects of the GDK are not optimized.
+We are releasing the GDK in [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) so we can react to feedback and iterate on development quickly. To facilitate this, during our alpha stage we don't have a formal deprecation cycle for APIs and workflows. This means that everything and anything can change. In addition, documentation is limited and some aspects of the GDK are not optimized.
 
-Given this, we don’t recommend using the the GDK for Unreal for games in the production phase just yet.
+Given this, for now we recommend using the GDK in projects in early production or prototype stage. This would ensure that your project's requirements are in line with the GDK's timeline.
```

Comment:
Add "the", so "projects in the early production or prototype stage"

Also "this would ensure" -> "this ensures"

Reply:
Added in the GDocs